Orange/classification/simple_tree.py

```diff
@@ -93,9 +93,9 @@ def fit_storage(self, data):
 
 class SimpleTreeModel(Model):
     def __init__(self, learner, data):
-        X = np.ascontiguousarray(data.X)
-        Y = np.ascontiguousarray(data.Y)
-        W = np.ascontiguousarray(data.W)
+        X = np.ascontiguousarray(data.X.astype(np.float64))
+        Y = np.ascontiguousarray(data.Y.astype(np.float64))
+        W = np.ascontiguousarray(data.weights.astype(np.float64))
```

Comment:
Because it was there before because
```python
In[20]: i = Table('iris')
In[21]: i.values.flags
Out[21]: 
  C_CONTIGUOUS : False
  F_CONTIGUOUS : True
  OWNDATA : False
  WRITEABLE : True
  ALIGNED : True
  UPDATEIFCOPY : False
In[22]: np.ascontiguousarray(i.values).flags
Out[22]: 
  C_CONTIGUOUS : True
  F_CONTIGUOUS : False
  OWNDATA : True
  WRITEABLE : True
  ALIGNED : True
  UPDATEIFCOPY : False
```
I don't dare presume what this was needed for. Also, weights are not floats by default (because they were set with `t.set_weights(1)`) and see 8d8101a4658885d6d346d4a40782878cf6fd9ce9's message.